src/dbnode/storage/series/buffer.go

```diff
@@ -60,6 +60,10 @@ const (
 	// enabled to see if this is a sane number.
 	evictedTimesArraySize = 8
 	writableBucketVer     = 0
+
+	// NB: this is the value all points will be truncated to.
+	truncatedValue float64 = 0
```

Comment:
Maybe we don't set it to a specific value and just let whatever the client is writing be the value it sets?

Reply:
Fair; pulling this into configs as discussed